src/sass/app.scss

```diff
@@ -14,6 +14,7 @@
 
 // Components
 @import "components/Alert.scss";
+@import "components/SubmissionPageInfo.scss";
```

Comment:
We aren't using this right? That component takes the [`RefileWarning` styles](https://github.com/cfpb/hmda-platform-ui/blob/0d0cf9fdb65886ba6a215bc8bee54fb8e3138f15/src/js/components/SubmissionPageInfo.jsx#L6).